galadriel/tools/web3/onchain/solana/jupiter.py

```diff
@@ -180,12 +183,12 @@ async def swap(
 
 
 if __name__ == "__main__":
-    wallet = SolanaWallet(key_path="path/to/keypair.json")
+    wallet = SolanaWallet(key_path=os.getenv("SOLANA_KEY_PATH"))  # type: ignore
     swap_tool = SwapTokenTool(wallet)
     print(
         swap_tool.forward(
             "So11111111111111111111111111111111111111112",
-            "3iQL8BFS2vE7mww4ehAqQHAsbmRNCrPxizWAT2Zfyr9y",
+            "JUPyiwrYJFskUPiHa7hkeR8VUtAeFoSYbKedZNsDvCN",
```

Comment:
Looks like accidental change from testing?

Reply:
Yes the previous one doesn't work. this one is the Jupiter official token, just for testing